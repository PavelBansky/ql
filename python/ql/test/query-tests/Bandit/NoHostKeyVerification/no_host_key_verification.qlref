Bandit/no_host_key_verification.ql