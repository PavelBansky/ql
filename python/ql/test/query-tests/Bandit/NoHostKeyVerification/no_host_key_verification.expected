| Test.py:4:40:4:59 | Attribute | Paramiko call with policy set to automatically trust the unknown host key. |
| Test.py:5:40:5:59 | Attribute | Paramiko call with policy set to automatically trust the unknown host key. |
